Library/RemoteConfig/RemoteConfigFeature+Helpers.swift

```diff
@@ -1,10 +1,12 @@
 /// Return remote config values either a value from the cloud, if it found one, or a default value based on the provided key
+private func featureEnabled(feature: RemoteConfigFeature, defaultValue: Bool = false) -> Bool {
+  let valueFromDefaults = AppEnvironment.current.userDefaults
+    .remoteConfigFeatureFlags[feature.rawValue]
 
-private func featureEnabled(feature: RemoteConfigFeature) -> Bool {
-  return AppEnvironment.current.userDefaults
-    .remoteConfigFeatureFlags[feature.rawValue] ??
-    (AppEnvironment.current.remoteConfigClient?
-      .isFeatureEnabled(featureKey: feature) ?? false)
+  let valueFromRemoteConfig = AppEnvironment.current.remoteConfigClient?
+    .isFeatureEnabled(featureKey: feature)
```

Comment:
nit: Unlike before, `valueFromRemoteConfig` will now be calculated each time. I wouldn't expect that to be a problem (it's probably cached in the remote config client and thus not an expensive operation) but just in case it is actually expensive or always does a network call or something, I think we should stick with early returns. Ex use `if let` blocks with returns instead of just having one return statement at the end.

Reply:
Ah, good point - will fix then merge.